ui/segmented-bar.reel/segment.reel/segment.css

```diff
@@ -0,0 +1,13 @@
+/* horizontal */
+.montage-Segment {
+    height: 100%;
+    box-sizing: border-box;
+    border: 1px solid black;
+    background: darkgray;
+}
+
+/* vertical, not used, set in JS */
+/*@media (max-width: 1000px) {*/
+/*.montage-Segment {*/
+/*}*/
+/*}*/
```

Comment:
If code's not used, I'd rather not see it. Git history's a good place to put stuff you may use later (as long as you don't lose the history), or pastebin, or somewhere else. Think of code as a story or an academic paper: You wouldn't send a story or paper for review, or publish it, with sections commented out and your scratch work included. If you had a question you'd like to raise for the reviewers, you might make a note of it, and the equivalent here would be a single line comment with the question. Commenting out code just makes it invisible to the computer, not to readers, and the secret of beautiful code is to write it to be read and only secondarily be to be executed.